server/lib/data_import/my_anime_list.rb

```diff
@@ -31,15 +32,28 @@ def get_media(external_id) # anime/1234 or manga/1234
       end
     end
 
+    def get_character(external_id)
+      media = Mapping.lookup('myanimelist', external_id)
```

Comment:
This should probably be `"character/#{external_id}"`, since it's clear from `get_character` what it gets.